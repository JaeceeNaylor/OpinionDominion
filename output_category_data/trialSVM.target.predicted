SentenceID: 1016296:2
Category: RESTAURANT#GENERAL

