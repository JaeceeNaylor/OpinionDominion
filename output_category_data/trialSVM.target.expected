SentenceID: 1016296:2
Category: SERVICE#GENERAL

